### PR TITLE
Sync active Azure env with TF

### PR DESCRIPTION
**Pull request by JosiahSiegel**
This PR updates TF to match the current Azure environment.

*Note for reviewers*
Please find anything you can to improve this since it is the foundation and you may not have any easier time to update!

### CODE REVIEW

**Comment by JosiahSiegel** [comment]
Added 3 commits that created a log analytics workspace, connected both function apps, service plan, and vnet and finally connected to app insights.

File: operations/app/terraform/modules/network/main.tf [hunk 1]
```diff
@@ -6,22 +6,22 @@
  */
 
 resource "azurerm_subnet" "cdc_app_subnet" {
-  name                 = var.app_subnet_name
-  resource_group_name  = var.resource_group
-  virtual_network_name = var.cdc_vnet_name
-  address_prefixes     = ["172.17.9.64/28"]
-  enforce_private_link_endpoint_network_policies = true  # true = disable, false = enable; see: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/subnet
-  service_endpoints    = [
+  name                                           = var.app_subnet_name
```
Comment (cwinters-usds) [inline operations/app/terraform/modules/network/main.tf:9]: Is there a setting for associating this subnet with the routing table? I made that change manually in the UI.

File: operations/app/terraform/modules/storage/main.tf [hunk 2]
```diff
@@ -65,37 +63,33 @@ resource "azurerm_storage_account" "storage_account" {
...
-  type                = "storage_account_blob"
-  resource_group      = var.resource_group
-  location            = var.location
+/* Generate multiple storage private endpoints via for_each */
+module "storageaccount_private_endpoint" {
+  for_each = toset(["blob", "file", "queue"])
+  source   = "../common/private_sa_endpoint"
+  primary = {
+    name           = "${azurerm_storage_account.storage_account.name}-${each.key}-privateendpoint"
+    type           = "storage_account_blob"
```
Comment (cwinters-usds) [inline operations/app/terraform/modules/storage/main.tf:72]: should `_blob` be `_${each.key}` instead?

**Review by cwinters-usds — APPROVED** [review]
just a couple questions but LGTM -- thanks for normalizing things and creating useful abstractions for the privatelinks etc :tada:

File: operations/app/terraform/modules/network/main.tf (hunk 1 above)
Comment (JosiahSiegel) [reply operations/app/terraform/modules/network/main.tf:9]: added

**Pull request merged**